bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -132,7 +134,15 @@ spec:
         lifecycle:
           preStop:
             exec:
-              command: ["/usr/share/openvswitch/scripts/ovs-ctl", "stop"]
+              command:
+              - /bin/bash
+              - -c
+              - |
+                #!/bin/bash
+                set -e
```

Comment:
I have my doubts about this...
$ help set | grep -- -e
      -e  Exit immediately if a command exits with a non-zero status.

So  in other words if ```kill -9 $(cat /var/run/openvswitch/ovs-vswitchd.pid)``` fails (for instance, if that PID wouldn't exist anymore for watherever reason) then we would never execute the ```kill $(cat /var/run/openvswitch/ovsdb-server.pid)```

I'm not sure if this is the behavior we want...

Reply:
Fixed. Did || true where the existing script had -e and removed the -e for the preStop hook because failure there will just kill the container later anyway.